new-readme.md

```diff
@@ -40,39 +41,40 @@ With reSolve you can easy and fast create different level difficulty application
 
 ### <a name="installation">Installation</a>
 
-We have a special package [create-resolve-app](packages/create-resolve-app) for creation of any reSolve based application. It can be one of the our examples, or any your application. By default `create-resolve-app` creates empty [Hello world](https://github.com/reimagined/resolve/tree/master/examples/hello-world) application. You can start working with new reSolve application just in **2 minutes**.
+The [create-resolve-app](packages/create-resolve-app) package is used to create a **reSolve**-based application. You can use it to get one of the examples, or make an empty application. By default, `create-resolve-app` generates an empty [Hello World](https://github.com/reimagined/resolve/tree/master/examples/hello-world) application. You can start working on your new **reSolve** application in **2 minutes**.
```

Comment:
Information about 2 minutes is only for marketing, so it should have some emotions, 'just' word, and '!' sign.

Reply:
We need to add a comment for corrector to this place in this case

Reply:
Why?

Reply:
Because Dirk removes such things in 100% cases. I'll try to convince him that it's required here and in the list on top

Reply:
We'll return this text. Main readme - it's not only documentation, it's our main marketing text. We have no site, only this md.

Reply:
Got it